### ISSUE TITLE
Missing semicolons after @include/@extend directives are not counted as lint

### ISSUE THREAD

**Issue by sndrs** [brigade/scss-lint#188]
This means that both of these pass:

``` scss
@mixin no-semicolon() {}
el {
  @include no-semicolon
}

%no-semicolon
el {
  @extend %no-semicolon
}
```

Should these fail, like property values would?
